share/wake/lib/system/remote_cache_runner.wake

```diff
@@ -337,7 +337,7 @@ def mkPostJobRequest ((RunnerInput label cmd vis env dir stdin _res _prefix _ is
     obytes
 
 # Creates a CacheAllowedRequest from the various inputs and outputs of a runner
-def mkCacheAllowedRequest ((RunnerInput label cmd vis env dir stdin _res _prefix _ isAtty _ _): RunnerInput) (output: RunnerOutput) (hidden: String) =
+def mkCacheAllowedRequest ((RunnerInput label cmd vis _ env dir stdin _res _prefix _ isAtty _ _): RunnerInput) (output: RunnerOutput) (hidden: String) =
     def Usage status runtime cputime mem _ibytes obytes = output.getRunnerOutputUsage
 
     CacheAllowedRequest label cmd dir env hidden isAtty stdin vis status runtime cputime mem obytes
```

Comment:
Most of the existing runners seem fine enough ignoring the `modifiable` inputs.  This along with `mkJobCacheRunner` are the pair I'm most concerned about.  So long as the invariant that the same outputs are produced from the same inputs holds up, it *should* be fine to cache, but it feels a bit too easy to have a buggy/malicious artifact slip through and poison the cache.